libc/src/string/memory_utils/generic/builtin.h

```diff
@@ -10,16 +10,16 @@
 #define LLVM_LIBC_SRC_STRING_MEMORY_UTILS_GENERIC_BUILTIN_H
 
 #include "src/__support/macros/attributes.h" // LIBC_INLINE
-#include "src/__support/macros/config.h"     // LIBC_HAS_BUILTIN
```

Comment:
The changes in this PR allow us to remove the dependency on `libc.src.__support.macros.config` from `memory_utils` in libc/src/string/memory_utils/CMakeLists.txt and `:__support_macros_config` from `string_memory_utils` in utils/bazel/llvm-project-overlay/libc/BUILD.bazel.

Reply:
Done.